packages/brave-browser-beta/brave-browser-beta.pacscript

```diff
@@ -0,0 +1,7 @@
+name="brave-browser-beta"
+pkgname="brave-browser-beta"
+version="1.27.81"
+url="https://github.com/brave/brave-browser/releases/download/v${version}/brave-browser-beta_${version}_amd64.deb"
+gives="brave-browser-beta"
```

Comment:
```suggestion

```
And this?

Reply:
I'm currently using it to remove deb packages using apt